clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -145,6 +146,7 @@ func (m *clusterOOMsTrackerMeasurement) initFields(config *measurement.Config) e
 		Namespace:     metav1.NamespaceAll,
 	}
 	m.msgRegex = oomEventMsgRegex
+	m.resourceVersionRecorded = make(map[string]bool)
```

Comment:
Do we need a map?
Is there a guarantee that events will be delivered with incrementing RV, so that keeping last seen RV would be enough?

Reply:
If so - why not compare with the last Event in ooms[] slice, that would be enough with O(1) access.

Reply:
> Is there a guarantee that events will be delivered with incrementing RV, so that keeping last seen RV would be enough?

Not so sure about that, from [K8s docs](https://kubernetes.io/docs/reference/using-api/api-concepts/#the-resourceversion-parameter):

>For example, clients must not assume resource versions are numeric, and may only compare two resource version for equality (i.e. must not compare resource versions for greater-than or less-than relationships).

That said, we might just save the last registered resource version and check it for equality whenever we meet a new OOM event in `handleOOMEvent` function.

Reply:
In general, we're sending events in the order. The exceptions are events from resync (which we explicitly disable setting period to zero) and the events coming from relist (which we actually observe).

So while the map is not the most efficient implementation, I actually think the map is the simplest one and we can proceed with it.